ui/js/models/perf/issue_tracker.js

```diff
@@ -0,0 +1,21 @@
+import treeherder from '../../treeherder';
+import { getApiUrl } from "../../../helpers/urlHelper";
+
+treeherder.factory('PhIssueTracker', [
```

Comment:
Thanks for the extra feedback, I wasn't sure if I should ask you or @camd about whether there weren't new ways we should be doing this. Are there any examples of using fetch elsewhere in the codebase we could use as an example?

Reply:
We unfortunately don't have many instances to copy from at present, other than say:
https://github.com/mozilla/treeherder/blob/cca48d14df73d470a59f79b8c1a4991c93a0da0b/ui/js/auth/AuthService.js#L13-L33

I think in a month or two the situation will be quite different however, so perhaps carrying on as-is is fine until then wrt fetch? Though for constants and similar it would be good to start using ES6 imports now instead of AngularJS constants/values/...

Perhaps something useful in the meantime would be if @ionutgoldan you could take a took at some of the React getting started guides etc (if you've not used it before), to reduce the spin up time in the future?

Reply:
@edmorely So basically we can merge this as-is for now.
Regarding learning React (I only did a tutorial on codecademy last year): with what will this help? Are you thinking of replacing Angular with it? Or maybe mix them together, using the better features of React?

Reply:
The rest of Treeherder is moving from AngularJS (1.x; the ancient version that's not forward compatible with Angular 2-6) to React. Perfherder will need to follow suit, though there's a chance we'll be able to help with the transition.

Reply:
@edmorely Did you file bugs for doing refactor work like the one you mentioned? If yes, I could pick them occasionally and implement.

Reply:
ie: Even if @camd is able to help with (or perform entirely) the Perfherder transition, it will still be useful for those working on Perfherder to know React, since it's what it will be written in, in the future.

Reply:
Yep, I totally agree. Funny thing here: a couple of weeks ago I was thinking of proposing myself for switching Perfherder to React. Development & performance wise is just better. These are good news for me :).

Reply:
Ah great! :-)

Btw elsewhere we're using angular2react to allow converting parts gradually:
https://github.com/coatue-oss/angular2react

Reply:
> There is currently only one meta bug for Perfherder under that:
https://bugzilla.mozilla.org/show_bug.cgi?id=1450044
>
> ...perhaps a good first step is to identify suitable sections for piece by piece conversion and file sub-bugs for those? Maybe @wlach has some ideas for how it might make sense to split up the work? :-)

@wlach If you find the time to file bugs against that meta, it would be great. After I re-ramp up with React, I will start picking them and proposing other bugs myself. Do you have the time for doing this? Otherwise I'll contact you when you're available for sync.

Reply:
Perhaps we can schedule a meeting between all of us, where @camd can advise as to what strategies for piecemeal conversion have worked well so far, and @wlach can help with overall Perfherder architecture background, and following that we can file the first batch of bugs? At the moment I don't think any one of us has enough context of the whole picture to be able to do so :-)

Reply:
@edmorley Yes, please create a meeting! I've actually been doing quite a bit of react code in the last year, so can probably help review the conversion (even if I don't have time to do much implementation).

Reply:
@edmorley Count me in!